Tools/Scripts/webkitpy/common/system/executive.py

```diff
@@ -226,9 +226,14 @@ def kill_process(self, pid):
                     return
                 else:
                     raise
-
-            # Give processes one chance to clean up quickly before exiting.
-            current_signal = signal.SIGKILL
+ 
+            # Fallback to SIGKILL before exiting.
+            if retries_left <= 0 and current_signal != signal.SIGKILL:
+                _log.error('Couldn\'t quit {} with SIGTERM, sending SIGKILL.'.format(pid))
+                current_signal = signal.SIGKILL
+                retries_left += 1
```

Comment:
You should explicitly set `retries_left` to `1`, this is the sort of logic that can bit us in the future if someone, for example, tries to make the retry counter go down `n` times faster in the future.